app/views/users/show.html.erb

```diff
@@ -0,0 +1,43 @@
+<h1>
+ <%= @user.username %>
+</h1>
+
+<h2>Own photos</h2>
+
+<ul>
+<% @user.own_photos.each do |photo| %>
+<div class="row mb-4">
+  <div class="col-md-6 offset-md-3">
+    <div class="card">
+      <%= image_tag photo.image, class: "card-img-top" %>
+      <div class="card-body">
+        <h5 class="card-title"><%= photo.owner.username %></h5>
+        <p class="card-text"><%= photo.caption %></p>
+      </div>
+      <ul class="list-group list-group-flush">
+        <% photo.comments.each { |comment| %>
+          <li class="list-group-item">
+            <div class="d-flex">
+              <div class="flex-shrink-0">
+                <img src="" alt="">
```

Comment:
If we dont use the image tag, its good practice to either remove it or add default values so it doesnt mess with the accessibility of the app :)